churner/tests/churner_test.go

```diff
@@ -91,39 +89,14 @@ func teardown() {
 func TestChurner(t *testing.T) {
 	ctx := context.Background()
 
-	// get the operator to churn in's transactor and bls public key registered
-	op := testConfig.Operators[0]
-	operatorTransactor, err := createTransactorFromScratch(
-		op.NODE_PRIVATE_KEY,
-		testConfig.EigenDA.OperatorStateRetreiver,
-		testConfig.EigenDA.ServiceManager,
-		logger,
-	)
-	assert.NoError(t, err)
-
 	keyPair, err := dacore.GenRandomBlsKeys()
 	assert.NoError(t, err)
 
-	quorumIds_ := make([]uint8, len(quorumIds))
-	for i, q := range quorumIds {
-		quorumIds_[i] = uint8(q)
-	}
-
-	operatorSalt := [32]byte{}
-	_, err = rand.Read(operatorSalt[:])
-	assert.NoError(t, err)
-
-	expiry := big.NewInt(1000)
-	privKey, err := crypto.GenerateKey()
-	assert.NoError(t, err)
-
-	err = operatorTransactor.RegisterOperator(ctx, keyPair, "socket", quorumIds_, privKey, operatorSalt, expiry)
```

Comment:
This used to be registering the public key but this isn't a part of the flow anymore.

Reply:
It looks v2 contracts are yielding benefits